docs/release_notes/20240129.mdx

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: 2024-01-29 LangSmith v0.2
+sidebar_position: 1
+---
+
+# Langsmith Release Notes
+
+## Week of 1/29 - LangSmith v0.2
+
+LangSmith 0.2 improves performance and reliability, adds a updated interface for reviewing trace data, and adds support for batch processing of traces.
+
+### Requirements
+
+- This release requires `langsmith-sdk` version ≥ `0.0.71` (Python) and  ≥ `0.0.56` (JS/TS) to support changes in pagination of API results.  Older versions will return an http status 405 error.
+
+### Breaking changes
+
+- The search syntax for metadata in runs has changed and limits support for nested JSON to a single level.  If you are supplying custom metadata in traces, you should flatten your metadata structure in order to allow it to be searchable, (e.g. `{"user_id": ..., "user_name":...,}`) and then search using `has(metadata, '{"user_name": ...}')`
+
+### LangSmith UEX Changes
+
+**Tracing** 
+
+- We refreshed the run tree so that the most important calls are displayed in the default view with the ability to show all steps in an expanded view.
+- Better rendering for tools, function calls, and documents in the trace view and playground.
+- Keyboard shortcuts (J, K) to navigate to the previous or next run while in the trace history view.
+- Trace histories now have infinite scroll, so you can get more context.
+- Trace to playground access for all LLM types.
+
+**Monitoring**
+
+- Drill downs and dynamic stat calculations by tag or metadata on the monitoring tab, so you can filter charts by a segment.
+
+### LangSmith SDK Changes
+
+**LangSmith JS SDK**
+
+- This release adds support for batch ingest of traces and runs to reduce ingest latency and the number of API calls to the Langsmith service.  This feature is currently in pre-release for `langchain` and `langsmith-sdk`.
+- Support for `runOverDataset` in JS to allow for bulk evaluation of your app over a reference dataset in LangSmith.
+
+### Performance and Reliability Changes
+
+- Improved performance when ingesting traces, reducing the delay between ingest and display in the LangSmith UI.
+- Improved performance for updates and deletes on annotation labels.
+- Added pagination of API responses.
+- Fixed an issue impacting natural language searches.
+
+### Admin changes
+
+- None
+
+## Notes for Self-Hosted Deployments
+
+### Infrastructure Changes
+
+- Added the `clickhouse` database service.  Run results will now be stored in ClickHouse instead of Postgres to improve performance and scalability and reduce delays in the time it takes for runs to appear in LangSmith.
+    - Note that if you wish to retain access to run data in the Langsmith UI after updating, a data migration will need to be performed.  Details are available at https://github.com/langchain-ai/helm/blob/main/charts/langsmith/docs/UPGRADE-0.2.x.md
+
+### Admin changes
```

Comment:
the duplicate of admin changes header is a bit confusing but cant think of a better way haha, maybe call it Self-Hosted Admin changes?